### PR TITLE
Layer Switcher Styling & Simplified user interaction

### PR DESCRIPTION
**Pull request by martyrion**
- I have styled the layer switcher control so that it resembles the compass and coordinates control (I think all controls that involve buttons etc should be styled the same way).
- I have removed the center button because the UI was really cluttered by two buttons side-by-side. I have attached this functionality to a double-click on the label. (maybe single click would do also).

I think it looks much nicer and homogenous now.

This is my first pull request ever so please let me know of any mistakes in the procedure.

https://user-images.githubusercontent.com/63229987/184296157-8cfb7675-f400-4fd3-8c24-f4e6a581abb9.mov

### CODE REVIEW

File: src/og/control/LayerSwitcher.js [hunk 1]
```diff
@@ -57,13 +57,14 @@ class LayerSwitcher extends Control {
...
         var lineDiv = document.createElement("div");
-
+        lineDiv.className = "layersEntry"
         var that = this;
-        var center = document.createElement("div");
-        center.classList.add("ogViewExtentBtn");
-        center.onclick = function () {
-            that.planet.flyExtent(obj.getExtent());
-        };
+        // var center = document.createElement("div");
```
Comment (Zemledelec) [inline src/og/control/LayerSwitcher.js]: Please, cleanup useless comments

**Review by Zemledelec — CHANGES_REQUESTED** [review]
Looks very nice! Please, cleanup comments.

File: src/og/ellipsoid/wgs84.js [hunk 2]
```diff
@@ -10,4 +10,4 @@ import { Ellipsoid } from "./Ellipsoid.js";
  * WGS84 ellipsoid object.
  * @type {Ellipsoid}
  */
-export const wgs84 = new Ellipsoid(6378137.0, 6356752.3142);
+export const wgs84 = new Ellipsoid(6378137.0, 6356752.3142); // Equitorial Radius (m), Polar Radius (m)
```
Comment (Zemledelec) [inline src/og/ellipsoid/wgs84.js:13]: This comment is ok:)

Comment (martyrion) [reply src/og/ellipsoid/wgs84.js:13]: Thanks. I just learned what is an ellipsoid 2 days ago! :) Thought it my be useful for some else

**Comment by martyrion** [comment]
I was wondering if it would be nice to have some other controls for drawing markers, lines, circles.
And also style the ruler control the same way - with a button

**Comment by Zemledelec** [comment]
> 

Most of the features are coming with needs, such as the ruler and layer animation controls. Sometimes for fun:) Drawing tools are very appreciated (3D especially), and this is not an easy thing to develop, especially if you want them to work nicely. In such a case welcome to create an issue and discuss it first.

**Pull request merged**